templates/realtime.html

```diff
@@ -1,6 +1,8 @@
 {% extends "layout.html" %}
 {% block head %}
 	{{ super() }}
+    <script src="node_modules/gridstack/dist/gridstack-h5.js"></script>
+    <link href="node_modules/gridstack/dist/gridstack.min.css" rel="stylesheet"/>
```

Comment:
The node_modules directory would be for if we were running a NodeJS server instead of Flask. For us these files should be placed in the static directory.